pkg/kapis/network/v1alpha2/handler.go

```diff
@@ -0,0 +1,77 @@
+package v1alpha2
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+
+	restful "github.com/emicklei/go-restful"
+)
+
+const ScopeQueryUrl = "http://weave-scope-app.weave.svc/api/topology/services"
```

Comment:
This means weave-scope is required for network topology, but how to enable and disable it? And we need to provide API for console to find out which components are enabled, check `pkg/apiserver/config`